charts/dbildungscloud-iam-keycloak/values.yaml

```diff
@@ -16,6 +16,18 @@ database:
 
 namespace: ""
 
+auth:
+  # existingSecret: Refers to a secret already present in the cluster, which is required for the authentication and configuration of the database setup tasks.
+  existingSecret: ""
+  name: keycloak
+  admin_password: ""
+  admin_user: ""
+  db_host: ""
```

Comment:
I think the database credentials would make more sense under database

Reply:
moved them under database